docs/rules/jsx-sort-default-props.md

```diff
@@ -0,0 +1,97 @@
+# Enforce propTypes declarations alphabetical sorting (react/jsx-sort-default-props)
+
+Some developers prefer to sort propTypes declarations alphabetically to be able to find necessary declaration easier at the later time. Others feel that it adds complexity and becomes burden to maintain.
+
+## Rule Details
+
+This rule checks all components and verifies that all defaultProps declarations are sorted alphabetically. A spread attribute resets the verification. The default configuration of the rule is case-sensitive.
```

Comment:
Can you elaborate about the spread attribute?

I would expect that sorts do not cross a spread boundary, but `d, a, ...obj, c, b` would be sorted into `a, d, ...obj, b, c`